release/changelog.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -ex
+
+source CONFIG
+source ${work}/common.sh
+
+export GOPATH=${dir}/go
+export PATH=$GOPATH/bin:$PATH
+
+echo "# CHANGELOG"
+echo
+echo "## v${ver}"
+echo
+
+for r in ${repos}; do
+    echo "[[$r](https://github.com/revel/$r)]"
+    echo
+    lines=$(git --no-pager -C ${src}/${r} log --oneline master..develop)
```

Comment:
yeah, this was just a first pass, I planned on integration with the generator, just needed to get things working first.